apps/voucher/graphql/schema.ts

```diff
@@ -0,0 +1,72 @@
+const typeDefs = `#graphql
+
+enum Status {
+  ACTIVE
+  PENDING 
+  PAID
+}
+
+type WithdrawLink {
+  id: ID!
+  userId: ID!
+  status: Status!
+  voucherAmountInCents: Float!
+  createdAt: String!
+  paidAt: String
+  salesAmountInCents: Float!
+  commissionPercentage: Float!
+  identifierCode: String!
+}
+
+type WithdrawLinkWithSecret {
+  id: ID!
+  userId: ID!
+  status: Status!
+  voucherAmountInCents: Float!
+  createdAt: String!
+  paidAt: String
+  salesAmountInCents: Float!
+  identifierCode: String!
+  voucherSecret: String!
+  commissionPercentage: Float!
+  uniqueHash: String!
+}
+
+type WithdrawLinksByUserIdResult {
+  withdrawLinks: [WithdrawLink!]!
+  totalLinks: Int
```

Comment:
I don't think there is a need for that unless we do pagination. 
if we do pagination, we should use the same pattern as the backend

Reply:
Removed pagination for now.